sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -129,26 +129,24 @@ def _create_checkpoint_entity(cls, checkpoint):
         Create a dictionary with `checkpoint` attributes.
         """
         checkpoint_entity = {
-            "PartitionKey": "{} {} {} Checkpoint".format(
+            "PartitionKey": u"{} {} {} Checkpoint".format(
                 checkpoint["fully_qualified_namespace"],
                 checkpoint["eventhub_name"],
                 checkpoint["consumer_group"],
             ),
-            "RowKey": checkpoint["partition_id"],
-            "offset": checkpoint["offset"],
-            "sequencenumber": checkpoint["sequence_number"],
+            "RowKey": u"{}".format(checkpoint["partition_id"]),
+            "offset": u"{}".format(checkpoint["offset"]),
+            "sequencenumber": u"{}".format(checkpoint["sequence_number"]),
         }
         return checkpoint_entity
 
     def _update_ownership(self, ownership, **kwargs):
         """_update_ownership mutates the passed in ownership."""
+        ownership_entity = TableCheckpointStore._create_ownership_entity(ownership)
         try:
-            ownership_entity = TableCheckpointStore._create_ownership_entity(ownership)
             metadata = self._table_client.update_entity(
                 mode=UpdateMode.REPLACE,
                 entity=ownership_entity,
-                etag=ownership["etag"],
-                match_condition=MatchConditions.IfNotModified,
```

Comment:
I'm not familiar with the table sdk..

do you know the context why those two are passed before but not passed now?
also is it something that JS is doing?

Reply:
good question, they were passed in before b/c etag was also passed which aligns with JS. Basically, if IfNotModified is passed in, the entity will only be updated if the etag matches/that entity hasn't been updated since. Otherwise, it will throw a ResourceModifiedError.

In JS, there is no match_condition equivalent, but if etag is passed in and there isn't an entity that matches, it will result in an error. I'm confirming with Chris that it's the same error, and if so, will add this back in.

Reply:
adding this back in